### PR TITLE
FS-855 Configuring frontend for COF fund and round

### PR DESCRIPTION
**Pull request by stevewalton28**

### CODE REVIEW

File: app/default/routes.py [hunk 1]
```diff
@@ -65,11 +68,12 @@ def new(account_id):
         url=f"{Config.APPLICATION_STORE_API_HOST}/applications",
         json={
             "account_id": account_id,
-            "round_id": request.form["round_id"] or Config.DEFAULT_ROUND_ID,
-            "fund_id": request.form["fund_id"] or Config.DEFAULT_FUND_ID,
+            "round_id": Config.DEFAULT_ROUND_ID,
+            "fund_id": Config.DEFAULT_FUND_ID,
```
Comment (gidsg) [inline app/default/routes.py:72]: Should we keep the previous fallback, which was designed to allow multiple funds passed by query string?

File: app/templates/tasklist.html [hunk 2]
```diff
@@ -9,8 +9,7 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">
-      {{application_meta_data["fund"]|kebab_case_to_human}}
-      application
+      Community ownership fund application
```
Comment (gidsg) [inline app/templates/tasklist.html:12]: Again could we retain the previous logic to make this more generic?

File: app/default/routes.py [hunk 3]
```diff
@@ -106,13 +111,20 @@ def tasklist(application_id):
         Config.GET_APPLICATION_ENDPOINT.format(application_id=application_id)
     )
     if not (application_response and application_response["sections"]):
-        return abort(404)
+        return abort(500)
```
Comment (gidsg) [inline app/default/routes.py:114]: Can we log something before we abort about what went wrong?

Comment (stevewalton28) [reply app/default/routes.py:114]: Yeh great idea I'll put that in

**Review by gidsg — APPROVED** [review]
LGTM ⛽ 👍

**Pull request merged**